init-openstack.html.md.erb

```diff
@@ -151,28 +151,36 @@ You must create and configure two Security Groups to restrict incoming network t
     $ git clone https://github.com/cloudfoundry/bosh-deployment
 
     # Fill below variables (replace example values) and deploy the Director
+    $ DIRECTOR_NAME='bosh-1'     # arbitrary name for the director
```

Comment:
@hoegaarden how about this format: https://github.com/cloudfoundry/docs-bosh/blob/master/init-vsphere.html.md.erb#L43 ?